### DISCUSSION TITLE
Connecter For Minecraft Servers?

### DISCUSSION THREAD

**Discussion by TouseefX** [Sinytra/Connector#677] [Ideas]
We have connector and also works with minecraft server. But problem is. If u don't have fabric mods that you mod at your client folder the server forces you to crash and if u have all mods same the server does it will don't crash, but I want a server with both fabric and forge to fix lags and troll people but the mods in client and server have to be same what if not we just created a new connector for Minecraft servers is that a great idea?

**Comment by Su5eD** [comment]
Connector works fine on servers, and is not required on the client

**Reply by TouseefX** [reply]
No it does not work even as I am on 47.1.3 minecraft server forge, and it keeps crashing, and sometimes it crashes on startup and some client mods that don't WORK WITH CLIENT works with server? IDK why but I try server mods and some of that don't work with server instead client I am saying you connector is broken for servers

No fog seems to crashes in startup because it does not load and try to crash server
and if u try to remove loading screen using mods it will stop whole thing
if you try to install custom minecraft animation like fresh animation it will crash but not for fabric, I don't know why
and most of PVP mods does not work with that, so rip PVP Mods
and also I see connector have some warning with server like client players problems
for some reason the boosting fps mod called lazy manage and lazydfu and memory lazy fix make server too fast that is faster than my client like how, and I don't even lag anymore that feels like I went from overheating to lazy Cool
that why we want connector on Servers
but biggest problem, all server host are not the same, so that leaves the question: What Is Best Server Host For Connector?

**Reply by coco875** [reply]
??? what do you mean ? all server host normally use minecraftForge by default so they are equal some to let you use plugin can use magma, mohist, etc but connector don't support hybrid server and yeah some mods can have issue so try identify what mod make the server crash to report it to connector

**Reply by TouseefX** [reply]
no forge was keep crashing unit i use magma server with forge mod and it sovles the porblem

**Reply by coco875** [reply]
ok and just for information you can use connector with more recent version of forge

**Comment by TouseefX** [comment]
idk for some reason sodium is working in my server?????????????????????????????????????????????????????????????????????

**Reply by coco875** [reply]
you need to remove embeddium/rubidium to work on server

**Reply by TouseefX** [reply]
i don't have that
